Search/Event/DeindexEvent.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/*
+ * This file is part of the MassiveSearchBundle
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Massive\Bundle\SearchBundle\Search\Event;
+
+use Symfony\Component\EventDispatcher\Event;
+
+/**
+ * Will be fired to deindex given subject.
+ */
+class DeindexEvent extends Event
+{
+    /**
+     * The object, which should be indexed.
+     *
+     * @var object
+     */
+    private $subject;
+
+    public function __construct($subject)
+    {
+        $this->subject = $subject;
+    }
+
+    /**
+     * @return object
```

Comment:
I think the comment here would be more important than on the variable, since this is the method the user uses.